plugins/modules/system/timezone.py

```diff
@@ -19,6 +19,8 @@
     For Linux it can use C(timedatectl) or edit C(/etc/sysconfig/clock) or C(/etc/timezone) and C(hwclock).
     On SmartOS, C(sm-set-timezone), for macOS, C(systemsetup), for BSD, C(/etc/localtime) is modified.
     On AIX, C(chtz) is used.
+  - Make sure that the zoneinfo files are installed with the approritate OS package, like C(tzdata) (usually always installed,
```

Comment:
Sorry, found a typo:
```suggestion
  - Make sure that the zoneinfo files are installed with the appropriate OS package, like C(tzdata) (usually always installed,
```